### PR TITLE
Add declarative text push button

### PR DESCRIPTION
**Pull request by KRLango**
The line plot display layer inspector section currently uses a text push button to create a cleaner UI. This will likely look far worse with normal pushbuttons here so I added a declarative wrapper to the text push button. This could also be used on the graphics "move to center" button.

### CODE REVIEW

File: nion/swift/Inspector.py [hunk 1]
```diff
@@ -4985,3 +4975,60 @@ def data_item_delete() -> None:
             return widget
 
         return None
+
+
+def make_line_plot_display_layer_button_row(document_controller: DocumentController.DocumentController,
```
Comment (Tiomat85) [inline nion/swift/Inspector.py:4980]: Double blank line before method.

**Review by Tiomat85 — CHANGES_REQUESTED** [review]
Generally fine, just one extra whitespace line.

**Review by lisham2000 — APPROVED** [review]
All looks good to me

File: nion/swift/Inspector.py (hunk 1 above)
Comment (KRLango) [reply nion/swift/Inspector.py:4980]: This is not a class method, so I think 2 blank lines is correct.

**Review by Ion-e — APPROVED** [review]
Draft looks good to me

**Review by Ion-e — APPROVED** [review]
Looks good, thanks!

**Review by cmeyer — CHANGES_REQUESTED** [review]
This looks good and I'm almost ready to merge it, but I think we should use a more qualified name "nionswift.text_push_button" instead of just "text_push_button" because I think we'll eventually move this code to `nionui` and I want to avoid the hassle of having two items registered with the same name while we migrate.

As long as you're making a change, adding tool tips to the buttons would be a nice addition.

**Comment by cmeyer** [comment]
Merged def9ad917301e71789a7d9e67faf37ee01da5394 333124a12a515c5f10bf93bec2149f4efe27d791

**Pull request closed without merging**